aait/web/group-3/starter_project_group3/app/layout.tsx

```diff
@@ -15,11 +16,10 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className='font-Montserrat'>
-        <Header/>
-        <ReduxProvider>
-          {children}
-        </ReduxProvider>
+      <body className="font-Montserrat">
```

Comment:
The header and Footer should also wrapped in the redux provider.

Reply:
sure.